plugins.json

```diff
@@ -144,6 +144,14 @@
       "author":"Kyle Fuller, Boris Bügling",
       "url":"https://github.com/CocoaPods/cocoapods-packager",
       "description":"Generate a framework or static library from a podspec."
+    },
+    {
+      "gem":"cocoapods-links",
+      "name":"Pod links",
```

Comment:
There seems to be a mixed convention in the plugin.json:

FooBar
Foo
Pod foo
Pod Foo
foo
Foo Bar
CocoaPods Foo

The last 2 follow the "CocoaPods Foo" syntax. I personally don't favor any which way so if CocoaPods Foo is preferred, I can switch it over